src/main/kotlin/com/katanox/tabour/integration/sqs/config/SqsConfiguration.kt

```diff
@@ -16,7 +16,7 @@ class SqsConfiguration(@Autowired val sqsProperties: SqsProperties) {
 
     @Bean
     @Primary
-    fun amazonSQSAsync(): SqsClient {
+    fun amazonSQSClient(): SqsClient {
```

Comment:
small question here.
Why don't we use the async client?

Reply:
We were not doing any async operation as we were using await, this client good for now, but sure we can provide another flow later on